README.md

```diff
@@ -33,6 +33,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 * [Python Culture](https://svn.python.org/projects/sandbox/trunk/dev/culture.rst) - python.org historic document
 * [TechnicalDebt](https://martinfowler.com/bliki/TechnicalDebt.html) - Martin Fowler
 * [Top 7 programmers bad habits](https://web.archive.org/web/20110803100310/http://www.makinggoodsoftware.com/2011/05/23/top-7-programmers-bad-habits) - Alberto Gutierrez
+* [When Good Enough Software is Best](https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=382191) - Edward Yourdon
```

Comment:
Does this need to be paid? 🤔

Reply:
I didn't check it sorry.
I was using my account from UFMG which I have access to the IEEE
:smile:

Reply:
I will close it then :)

Reply:
Thanks anyway for the intention! 😁

Reply:
Find this link on Google Scholar: https://www.liamwho.com/wp-content/uploads/When-Good-Enough-Software-Is-Best.pdf (

@marcelotrevisani, can you confirm if this is the same paper? Can you reopen the PR if this is the case?

Reply:
Yeap! It is the same article! :)

Reply:
I think I deleted this fork :/
I will create another PR for that then
Thanks @igortg  !